<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modelo300</class>
 <widget class="QDialog" name="modelo300">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modelo 303</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>NIF:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="niflineEdit">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Apellidos / razón social:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="apellidos_razonlineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nombre:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nombrelineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="redemecheckBox">
         <property name="text">
          <string>Registro Devolución Mensual</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="complgroupBox">
       <property name="title">
        <string>Complementaria / Sustitutiva</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QRadioButton" name="sustitutivaradioButton">
          <property name="text">
           <string>Sustitutiva</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="complementariaradioButton">
          <property name="text">
           <string>Complementaria</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Núm.declaración anterior:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="anteriorlineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Importe a compensar de períodos anteriores:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="compensarlineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="tipocomboBox">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>U - Domiciliación del ingreso en CCC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I - Ingreso</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C - Solicitud de compensación</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D - Devolución</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G - Cuenta corriente tributaria-ingreso</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N - Sin actividad/resultado cero</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>V - Cuenta corriente tributaria -devolución</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>Cuenta banco:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CtalineEdit">
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ctapushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="gen300pushButton">
       <property name="text">
        <string>&amp;Genera 303</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/aeat.png</normaloff>:/iconos/graf/aeat.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>niflineEdit</tabstop>
  <tabstop>apellidos_razonlineEdit</tabstop>
  <tabstop>nombrelineEdit</tabstop>
  <tabstop>complgroupBox</tabstop>
  <tabstop>sustitutivaradioButton</tabstop>
  <tabstop>complementariaradioButton</tabstop>
  <tabstop>anteriorlineEdit</tabstop>
  <tabstop>redemecheckBox</tabstop>
  <tabstop>compensarlineEdit</tabstop>
  <tabstop>tipocomboBox</tabstop>
  <tabstop>CtalineEdit</tabstop>
  <tabstop>ctapushButton</tabstop>
  <tabstop>gen300pushButton</tabstop>
  <tabstop>salirpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>modelo300</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gen300pushButton</sender>
   <signal>clicked()</signal>
   <receiver>modelo300</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
